core/src/main/java/com/linecorp/armeria/client/metric/MetricCollectingClient.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2016 LINE Corporation
```

Comment:
These are renamed from `DropwizardMetricCollecting*` so I retained the inception year.